raw_data/preprocessed/africa/guinea-bissau_portuguese.txt

```diff
@@ -0,0 +1,199 @@
+
+Luz e níveis de luminosidade
+(13) [1] Todos têm direito a um nível de luminosidade conveniente à sua saúde, bem-estar e conforto na habitação, no local de trabalho e nos espaços livres públicos de recreio, lazer e circulação. [2] O nível de luminosidade para qualquer lugar deve ser o mais consentâneo com vista ao equilíbrio dos ecossistemas transformados de que depende a qualidade de vida da população. [3] Os anúncios luminosos não devem perturbar o sossego, a saúde e o bem-estar das pessoas. [4] Fica condicionado, em legislação especial, o volume dos edifícios a construir que, pelo ensombramento dos espaços livres, públicos ou privados, prejudique a qualidade de vida das pessoas ou a vegetação.
+
+Solo
+(14) [1] A defesa e valorização do solo como recurso natural determina a adopção de medidas conducentes à sua racional utilização. [2] A ocupação e a utilização do solo para fins urbanos e industriais ou implantação de equipamentos e infraestruturas serão condicionadas pela sua natureza, topografia e recursos naturais dele dependentes.
+
+Subsolo
+(15) [1] A exploração dos recursos do subsolo deverá ter em conta: [a] os interesses de conservação da natureza e dos recursos naturais; [b] a necessidade de obedecer a um plano global de desenvolvimento e, portanto, a uma articulação a nível nacional; [c] os interesses e questões que local e mais directamente interessem às regiões, sectores e municípios onde se insiram. [2] Sem prejuízo do disposto no número anterior do presente artigo, a exploração do subsolo deverá ser orientada através da legislação especial.
+
+SECÇÃO II - COMPONENTES AMBIENTAIS HUMANOS
+
+Componentes
+(16) [1] Os componentes ambientais humanos definem, no seu conjunto, o quadro específico de vida, onde se insere e de que depende a actividade do homem. [2] São componentes ambientais humanos: [a] paisagem; [b] património cultural, natural e construído. [3] O Ordenamento do Território e a gestão urbanística terão em conta o disposto na presente lei.
+
+Paisagem
+(17) A implantação de construções ou outros empreendimentos será condicionado, nos termos de legislação especial, a fim de não provocar ”impacte” violento na paisagem pré-existente.
```

Comment:
The quotations around ”impacte” are the same as normal ("). But it looks like two backwards " instead of one backward and one forward. I think the characters are the same despite out of ordinary visuals, not sure if this matters.